tests/quantization/test_bitsandbytes.py

```diff
@@ -72,5 +77,7 @@ def test_load_bnb_model(vllm_runner) -> None:
             # compare the first line of the output
             actual_output = outputs[index][1][0].split('\n', 1)[0]
             expected_output = expected_outputs[index].split('\n', 1)[0]
+            #truncate actual_output
+            actual_output = actual_output[:len(expected_output)]
```

Comment:
Can you avoid this? What is the expected output is longer than actual output?

Reply:
I add an assert to make sure expected output is larger